src/opers.c

```diff
@@ -1823,61 +1824,25 @@ Obj FuncCOMPACT_TYPE_IDS( Obj self )
 {
   NextTypeID = -(1L << NR_SMALL_INT_BITS);
   CallbackForAllBags( FixTypeIDs );
+  CALL_0ARGS(FLUSH_ALL_METHOD_CACHES);
   return INTOBJ_INT(NextTypeID);
 }
 
 /****************************************************************************
 **
-*F  DoOperation( <name> ) . . . . . . . . . . . . . . .  make a new operation
+*F  DoOperation<N>Args( <oper>, ... ) . . . . . . . . . . .Operation Handlers
```

Comment:
Change ". .Operation" to ".  Operation" (i.e. replace the last period by a space)